distributions/test/itests/src/test/java/org/codice/ditto/replication/admin/query/ITReplicationQuery.java

```diff
@@ -37,178 +42,148 @@
 @DDFCommonOptions.ConfigureLogging
 @ReplicationOptions.Install
 @TestDelimiter(stdout = true, elapsed = true)
+@InstallBundle(bundle = @MavenUrl(groupId = "org.awaitility", artifactId = "awaitility"))
 @ServiceAdmin
 @ConfigurationAdmin
 @RunWith(Dominion.class)
 public class ITReplicationQuery {
 
-  private static final String NO_EXISTING_CONFIG = "NO_EXISTING_CONFIG";
-
   @Interpolate
   private static String GRAPHQL_ENDPOINT = "https://localhost:{port.https}/admin/hub/graphql";
 
-  private static final String URL = "https://localhost:9999/services";
+  private final SitesGraphQL sitesGraphql = new SitesGraphQL(GRAPHQL_ENDPOINT);
 
-  // ----------------------------------- Site Tests -----------------------------------//
+  private final ReplicationsGraphQL replicationsGraphql = new ReplicationsGraphQL(GRAPHQL_ENDPOINT);
 
-  @Test
```

Comment:
✏️ A lot of these tests I removed because they were testing error handling, which is already done at the unit test level.